spec/controllers/moderator_applications_controller_spec.rb

```diff
@@ -0,0 +1,172 @@
+# frozen_string_literal: true
+
+# The application ID to use in requests.
+def application_id
+  # @application.id will always be set unless :does_not_exist is set for the example,
+  # in which case we specifically want to use an id that does not exist.
+  @application.id || 192_139
```

Comment:
I don't like the idea of using a magic number here. I'm not sure what I should use instead, though.